.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+.DS_Store
+dist/
+*.orig
+.idea/
+bundle.js
+package-lock.json
```

Comment:
Just to make sure, have we agreed on removing "package-lock.json" from the set of tracked files? I am not against the idea, but I wanted to double-check (as its inclusion as part of the source files is usually recommended by the NPM team)

Reply:
We have had issues with package.lock.json (it gets corrupted from time to time :-( ), we should some day give a try though